modules/library/src/it/scala/zio/elasticsearch/IntegrationSpec.scala

```diff
@@ -27,8 +27,8 @@ import zio.test.{Assertion, Gen, TestAspect, ZIOSpecDefault, checkN}
 
 trait IntegrationSpec extends ZIOSpecDefault {
 
-  val elasticsearchLayer: ZLayer[Any, Throwable, ElasticExecutor] =
-    HttpClientZioBackend.layer() >>> ElasticExecutor.local
+  val elasticsearchLayer: ZLayer[Any, Throwable, Elasticsearch] =
```

Comment:
Can we use an alias for this? `ULayer`?

Reply:
Yeah, but I think we should use `TaskLayer`.